modules/atmosphere/rendering/atmospheredeferredcaster.h

```diff
@@ -75,6 +75,7 @@ class AtmosphereDeferredcaster : public Deferredcaster {
     void initializeCachedVariables(ghoul::opengl::ProgramObject& program) override;
 
     void update(const UpdateData&) override;
+    float eclipseShadow(glm::dvec3 position, bool ground);
```

Comment:
Maybe:
```suggestion
    float eclipseShadow(glm::dvec3 position, bool isGround);
```
Not sure what the `ground` parameter expresses